Makefile

```
@@ -43,6 +47,7 @@ help:
 	@echo "  docserve           run an HTTP server on the docs directory"
 	@echo "  html               make HTML docs"
 	@echo "  htmlview           open the index page built by the html target in your browser"
+	@echo "  htmllive           rebuild and reload HTML files in your browser"
```

```suggestion
	@echo "  htmllive           rebuild and reload HTML files in your browser"
	@echo "  htmlview           open the index page built by the html target in your browser"
```
This list is otherwise alphabetical.